packages/css-library/src/stylesheets/core.scss

```diff
@@ -1,6 +1,7 @@
 @use "../../dist/tokens/scss/variables" as *;
 
 @forward 'uswds-theme';
+@use 'uswds-fonts';
```

Comment:
I need to read up on what's going on under the hood for `@use` because I don't fully understand why moving this `@use` statement to here from `base/fonts.scss` would work. Any idea why?

Reply:
The difference is that in the core.scss it makes use of the "uswds-theme" settings, which includes the `$theme-font-path` variable (see attached screenshot).
![Screenshot 2025-01-29 at 8 59 35 AM](https://github.com/user-attachments/assets/e2d3c9de-49ca-43c6-8abe-9aed53b974ef)